lib/cli.js

```diff
@@ -94,7 +94,7 @@ optimist
     .describe('reporters', 'reporters to print out the results on stdout')
     .alias('reporters', 'r')
     .describe('suite', 'overwrites the specs attribute and runs the defined suite')
-    .describe('spec', 'run only a certain spec file - overrides specs piped from stdin')
+    .describe('spec', 'specifies spec file(s) or filter(s) for specs piped from stdin')
```

Comment:
I think this wording change makes it a little bit confusing. It looks like you only can pipe in the specs.

Reply:
I can try to spiff it up some.  Is there a limit to how long you want that to be?

Reply:
I tried to reword it slightly and removed the references to stdin, but maybe that's not entirely what you wanted.  I'm trying to convey that you can either specify the exact spec files you want to run or else you want to filter any spec files that have been defined elsewhere.  Suggestions appreciated.